src/testdrive/src/parser.rs

```diff
@@ -539,6 +562,7 @@ impl<'a> Iterator for LineReader<'a> {
         let mut line = String::new();
         let mut chars = self.inner.char_indices().fuse().peekable();
         while let Some((i, c)) = chars.next() {
+            self.raw_pos += 1;
```

Comment:
@def- this needs to be `+= i - previous_i` to account for non-ascii chars taking more than 1 byte